src/Calculator/Resources/en-US/Resources.resw

```diff
@@ -4723,4 +4723,24 @@
     <value>Use system setting</value>
     <comment>Lable for the app theme option to use system setting</comment>
   </data>
+  <data name="TitleBarBackButton.[using:Windows.UI.Xaml.Automation]AutomationProperties.Name" xml:space="preserve">
+    <value>Back</value>
+    <comment>Screen reader prompt for the Back button in title bar to back to main page</comment>
+  </data>
+  <data name="SettingsPageOpenedAnnouncement" xml:space="preserve">
+    <value>Settings Page</value>
+    <comment>Announcement used when Settings page is opened</comment>
+  </data>
+  <data name="AppThemeExpander.[using:Windows.UI.Xaml.Automation]AutomationProperties.Name" xml:space="preserve">
+    <value>App theme Setting</value>
+    <comment>Screen reader prompt for the App theme Setting radio buttons group</comment>
+  </data>
+  <data name="LightThemeRadioButton.[using:Windows.UI.Xaml.Automation]AutomationProperties.Name" xml:space="preserve">
+    <value>Light mode</value>
+    <comment>Screen reader prompt for the App theme Light mode radio button</comment>
+  </data>
+  <data name="DarkThemeRadioButton.[using:Windows.UI.Xaml.Automation]AutomationProperties.Name" xml:space="preserve">
+    <value>Dark mode</value>
+    <comment>Screen reader prompt for the App theme Dark mode radio button</comment>
+  </data>
```

Comment:
The narrator will announce the content of radio button and it seems not necessary to set the AutomationProperties.Name for narrator. Also, I find some typos as "expender" above like in line 4708 and 4712.